sdks/python/apache_beam/transforms/userstate.py

```diff
@@ -63,16 +63,30 @@ def to_runner_api(self, context):
 class CombiningValueStateSpec(StateSpec):
   """Specification for a user DoFn combining value state cell."""
 
-  def __init__(self, name, coder, combine_fn):
+  def __init__(self, name, coder=None, combine_fn=None):
```

Comment:
Can you please add pydoc to the constructor that explains how the parameters are handled?